README.md

```diff
@@ -1,72 +1,41 @@
 # Microsoft Azure Active Directory Authentication Library (ADAL) for Python
+=====================================
 
-The ADAL for python library makes it easy for python applications to authenticate to AAD in order to access AAD protected web resources.
+[![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python.svg?branch=dev)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-python)
+[![Documentation Status](https://readthedocs.org/projects/adal-python/badge/?version=latest)](https://adal-python.readthedocs.io/en/latest/?badge=latest)
 
-## Usage
+|[Getting Started](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki)| [Docs](https://aka.ms/aaddev)| [Samples](https://github.com/azure-samples?query=active-directory)| [Support](README.md#community-help-and-support)
+| --- | --- | --- | --- |
 
-### Install
```

Comment:
I understand the motivation of "Remove duplication. Moved removed content into wiki."
But do we consider keeping some most-seeking contents, such as "how to install", here in this de-facto project home page? In fact, we [fine tuned](https://github.com/AzureAD/azure-activedirectory-library-for-python/commits/nc-readme-updates/README.md) those installation instructions a couple times before, based on real-world supporting request. Better not lose those valuable knowledge and efforts.

Reply:
We are not losing the instructions recorded over time. They are just in the Wiki page now. I will link to it from the Readme here since it is a "most-seeked" content.

Reply:
Oh I see. Last time I checked, I tried find-by-eye and also "CTRL+F" (i.e. find in page) but could not find keyword "install" in our [Wiki home page](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki), its sidebar, or the [FAQ page](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/FAQ). I was not aware that "install" is inside the ["Basic"](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/ADAL-basics).

Now you have kept this "most-seeked" content by a link to that very wiki page. That is a reasonable solution! 👍

Reply:
Oh I see. Last time I checked, I tried find-by-eye and also "CTRL+F" (i.e. find in page) but could not find keyword "install" in our [Wiki home page](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki), its sidebar, or the [FAQ page](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/FAQ). I was not aware that "install" is inside the ["Basic"](https://github.com/AzureAD/azure-activedirectory-library-for-python/wiki/ADAL-basics).

Now you have kept this "most-seeked" content by a link to that very wiki page. That is a reasonable solution! 👍 

Closed.